CASSANDRA.md

```diff
@@ -153,9 +148,14 @@ object Phantom extends App {
 }
 ```
 
-Phantom requires mapping classes to lift the database model to DSL types. The query definition also requires special equality operators.
+Phantom requires mapping classes to lift the database model to DSL types. The query definition also requires special equality operators. The database model however adds fully type-safe schema compatible operations at compile time, meaning
+phantom will not allow you to produce invalid queries with respect to the Cassandra schema.
```

Comment:
@fwbrasil No, phantom manages the entire schema creation process, and even manages migrations for you. The mapping DSL is the "source of truth". It can check against the database like Quill does, but in reality it wants to manage the schema by design.